qa/tasks/cephfs/test_snap_schedules.py

```diff
@@ -120,14 +120,15 @@ def assert_if_not_verified(self):
 
     def verify(self, dir_path, max_trials):
         trials = 0
-        snap_path = "{0}/.snap".format(dir_path)
+        sdn = self.mount_a.client_config.get('client_snapdir', '.snap')
```

Comment:
Here I think you need to try `self.fs.config.get("client_snapdir")` first to distinguish whether this is from vstart_runner.py test ? And you can verify you new change and have a quick test locally.


It should be something like:

```
sdn = self.fs.config.get("client_snapdir", "")
if sdn == "":
  if kclient:
    sdn = self.mount_a.client_config.get('snapdirname', '.snap')
  else:
    sdn = self.mount_a.client_config.get('client_snapdir', '.snap')
```

Or you can add one helper instead to get the snapshot name and you need to do the same in other places.

Reply:
Okay.